src/main/scala/org/broadinstitute/dsde/workbench/sam/service/TosService.scala

```diff
@@ -55,6 +55,15 @@ class TosService(val directoryDao: DirectoryDAO, val tosConfig: TermsOfServiceCo
       )
     )
 
+  def getTosText(docSet: Set[String]): IO[Option[String]] =
+    docSet match {
+      case set if set.isEmpty => IO.pure(Option(termsOfServiceText))
+      case set if set.equals(Set("privacyPolicy")) => IO.pure(Option(privacyPolicyText))
+      case set if set.equals(Set("termsOfService")) => IO.pure(Option(termsOfServiceText))
+      case set if set.equals(Set("termsOfService", "privacyPolicy")) => IO.pure(Option(termsOfServiceText + "\n\n" + privacyPolicyText))
```

Comment:
Sorry, could we pull these out into constants and use them here and in the api routes? magic strings are making me nervous.